### PR TITLE
Calculating emissions

### PR DESCRIPTION
**Pull request by AleksanderRDL**
Merge? no? yes?

### CODE REVIEW

File: src/shared/calculateOptimalTime.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+#ifndef calculateOptimalTime_H
```
Comment (Emilorzz) [inline src/shared/calculateOptimalTime.h:1]: your define should be all caps

Comment (AleksanderRDL) [reply src/shared/calculateOptimalTime.h:1]: True

**Review by Emilorzz — APPROVED** [review]

File: src/shared/calculateOptimalTime.c [hunk 2]
```diff
@@ -0,0 +1,38 @@
+#include <stdio.h>
+#include <time.h>
+#include <string.h>
+#include "calculateOptimalTime.h"
+#include "DMIParser.h"
+#include "loadPseudoData.h"
```
Comment (Emilorzz) [inline src/shared/calculateOptimalTime.c:6]: oh woops, change `loadPseudoData.h` to `fileIO.h`

**Review by Aavild — APPROVED** [review]

**Review by Emilorzz — APPROVED** [review]

**Review by frederikja163 — CHANGES_REQUESTED** [review]
Some changes have been requested over discord

**Review by frederikja163 — APPROVED** [review]
Looks good, could have more comments and more thorough testing, but its fine for now.

**Review by Emilorzz — APPROVED** [review]

**Pull request merged**